### PR TITLE
adding public http import support

### PR DESCRIPTION
**Pull request by kcibul**

### CODE REVIEW

File: engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala [hunk 1]
```diff
@@ -479,9 +479,9 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
       case w: WorkflowSourceFilesWithDependenciesZip => validateNamespaceWithImports(w)
       case w: WorkflowSourceFilesWithoutImports =>
         val importResolvers: Seq[ImportResolver] = if (importLocalFilesystem) {
-          List(WdlNamespace.fileResolver)
+          List(WdlNamespace.fileResolver, WdlNamespace.httpResolver)
         } else {
-          List.empty
+          List(WdlNamespace.httpResolver)
```
Comment (cjllanwarne) [inline engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala]: So just to be clear, there's no way to turn this off?

Comment (kcibul) [reply engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala]: Yeah -- there's actually no way to turn any of these off really.  This whole section could do with a "configurability" ticket IMHO and streamline these code paths.  @katevoss future ticket?

Comment (geoffjentry) [reply engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala]: What is the argument for turning these off? 

security concerns?

Comment (cjllanwarne) [reply engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala]: Unforeseen security concerns would be my big one. Based on our history of other things that we thought should be on for everyone ending up with red faces and an off switch a few weeks later

**Pull request merged**